README.md

```diff
@@ -1,26 +1,27 @@
-<img width="80" src="https://raw.githubusercontent.com/donnisnoni95/v-logo/master/dist/v-logo.svg?sanitize=true">
+<p align="center">
+    <img width="80" src="https://raw.githubusercontent.com/donnisnoni95/v-logo/master/dist/v-logo.svg?sanitize=true">
+</p>
 
-# The V Programming Language
+<h1 align="center">The V Programming Language</h1>
 
-[![Build Status](https://github.com/vlang/v/workflows/CI/badge.svg)](https://github.com/vlang/v/commits/master)
-<a href='https://github.com/sponsors/medvednikov'><img src='https://camo.githubusercontent.com/da8bc40db5ed31e4b12660245535b5db67aa03ce/68747470733a2f2f696d672e736869656c64732e696f2f7374617469632f76313f6c6162656c3d53706f6e736f72266d6573736167653d254532253944254134266c6f676f3d476974487562'></a>
-<a href='https://patreon.com/vlang'><img src='https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fshieldsio-patreon.herokuapp.com%2Fvlang%2Fpledges&style=for-the-badge' height='20'></a>
-[![Twitter handle][]][Twitter badge]
-
-https://vlang.io
-
-Documentation: [vlang.io/docs](https://vlang.io/docs)
+<div align="center">
 
-Changelog: [github.com/vlang/v/blob/master/CHANGELOG.md](https://github.com/vlang/v/blob/master/CHANGELOG.md)
+[Website](https://vlang.io) |
+[Docs](https://vlang.io/docs) |
+[Changelog](https://github.com/vlang/v/blob/master/CHANGELOG.md) |
+[Speed](https://fast.vlang.io/)
 
-Speed Test: [fast.vlang.io](https://fast.vlang.io/) (monitors compilation speed for each commit to verify there are no speed regressions)
+</div>
 
-Twitter: [twitter.com/v_language](https://twitter.com/v_language)
+<div align="center">
 
-Discord (primary community): [discord.gg/vlang](https://discord.gg/vlang)
-
-Installing V: [github.com/vlang/v#installing-v-from-source](https://github.com/vlang/v#installing-v-from-source)
+[![Build Status](https://github.com/vlang/v/workflows/CI/badge.svg)](https://github.com/vlang/v/commits/master)
+[![Sponsor](https://camo.githubusercontent.com/da8bc40db5ed31e4b12660245535b5db67aa03ce/68747470733a2f2f696d672e736869656c64732e696f2f7374617469632f76313f6c6162656c3d53706f6e736f72266d6573736167653d254532253944254134266c6f676f3d476974487562)](https://github.com/sponsors/medvednikov)
+[![Patreon](https://img.shields.io/badge/patreon-donate-yellow.svg)](https://patreon.com/vlang)
+[![Twitter handle][]][Twitter badge]
+[![Discord](discord.gg/vlang)](https://img.shields.io/discord/592103645835821068?label=Discord)
```

Comment:
It should be `[![Alt title](URL to image)](URL to website, service, etc)`

[![Discord](https://img.shields.io/discord/592103645835821068?label=Discord)](discord.gg/vlang)
```
[![Discord](https://img.shields.io/discord/592103645835821068?label=Discord)](discord.gg/vlang)
```

Reply:
Also, it's better to do not skip protocol in URLs.